src/server/game/Entities/Player/Player.cpp

```diff
@@ -7274,6 +7275,14 @@ void Player::RewardReputation(Quest const* quest)
     }
 }
 
+void Player::RewardOtherBonusTalentPoints(uint32 bonusTalentPoints)
```

Comment:
I think a better name would be `RewardExtraBonusTalentPoints` then I think it's ok for me